### ISSUE TITLE
Добавить метод, предоставляющий информацию об API

### ISSUE THREAD

**Issue by ggguser** [gosdev/json-rpc#2]
Метод `rpc.describe` на 'specs' эндпоинте:
**Запрос**
```json
{
	"jsonrpc": "2.0",
	"id": "e67fd53e-beb7-4ae1-aa56-64af82f05ef3",
	"method": "rpc.describe"
}
```
**Ответ:**
```
{
    "jsonrpc": "2.0",
    "id": "e67fd53e-beb7-4ae1-aa56-64af82f05ef3",
    "result": {
        "name": "awesome-api",
        "project_id": "00e78501-65b0-4e19-b02d-d3b94c3685dd",
        "versions": ["1.1.1", "2.0.0"],
        "endpoints": ["api/jsonrpc", "self", "specs"]
    }
}
```

**Comment by axxilius** [comment]
Разверни, плиз, нафиг он нужен?

**Comment by ggguser** [comment]
> Разверни, плиз, нафиг он нужен?

@axxilius 
Для механизма discovery, например.
У тебя поднялось какое-то API на каком-то хосте. Как ты узнаешь, что это за апи?
Сейчас мы это делаем по лейблам докер контейнера.

Набор возвращаемых полей обсуждается.

Предлагаю добавить сюда же поле, с указанием зависимых сервисов.

Тогда возможен такой сценарий:
Есть три сервиса: продюсер, консьюмер и брокер.
Ты, как брокер по какому-то событию запрашивает версии продюсера.
Получив версии, брокер запрашивает схемы операций новой версии по эндпоинту {version}/specs.
Брокер знает, какие сервисы используют этого продюсера.
Брокер делает дифф между старой и новой схемой и выдать отчёт об изменившихся операциях. 

Для того, чтобы протестировать совместимость, нужно отдельно описывать в схеме запросы, которые отправляет консьюмер.

**Comment by axxilius** [comment]
Согласен, только тогда версионировать надо не API а весь сервис, так как в разных версиях могут меняться зависимости. Что-то типа
```json
{
    "jsonrpc": "2.0",
    "id": "e67fd53e-beb7-4ae1-aa56-64af82f05ef3",
    "result": {
        "name": "awesome-api",
        "project_id": "00e78501-65b0-4e19-b02d-d3b94c3685dd",
        "version": "1.5.12",
        "endpoints": {
            "public": "/api/jsonrpc", 
            "self": "/self", 
            "specs": "/specs",
            "metrics": "/metrics" 
        },
        "require": ["ugly-service:^1.2", "not-bad-staff:0.15"]
    }
}
```
Тогда действительно, service discivery сможет строить карту зависимостей и не добавлять в гейт операции того сервиса, для которого не удовлетворены все зависимости.
